components/centraldashboard/app/server.ts

```diff
@@ -14,10 +15,11 @@ async function main() {
   const app: express.Application = express();
   const k8sService = new KubernetesService(new KubeConfig());
   const metricsService = await getMetricsService(k8sService);
+  const profileController = new DefaultApi('http://kfam.kubeflow.svc.cluster.local/kfam/');
```

Comment:
Do we have to think about the situation where the user chooses to use a different namespace to deploy the core KF components to since it is possible with a CLI-based deployment? In that case, I think both the dashboard and kfam services will be deployed in the same namespace so in theory it should be possible to determine the namespace and use it.

Reply:
Got some clarity from chatting with Zhenghui on Metadata concerning the default namespace. It seems many assumptions are made about the default namespace being **kubeflow** so we may be OK with this for now but it might be worth adding a TODO or handling it dynamically if we find it's a trivial change.

Reply:
So if my understanding is correct from my convo with @abhi-g we should **always** have a `kubeflow` namespace (_which is almost like `system`_). If that's incorrect, I'll def add it in

Reply:
Yeah I think that's the assumption being made across all system components. I do believe it's possible to change the name of the default namespace to something else when deploying using the CLI method, but all bets are off. We're following this approach in the Metadata project so it seems reasonable to do so here as well.

Reply:
Fair enough, will do!